packages/ifpack2/src/Ifpack2_Relaxation_def.hpp

```diff
@@ -572,7 +574,7 @@ apply (const Tpetra::MultiVector<scalar_type, local_ordinal_type, global_ordinal
       }
     }
   }
-  ApplyTime_ += Time_->totalElapsedTime ();
+  ApplyTime_ += timer->totalElapsedTime ();
   ++NumApply_;
```

Comment:
We could actually get rid of `NumApply_` etc., since Teuchos::Time has a call count.